public/app/features/explore/LiveLogs.tsx

```diff
@@ -3,6 +3,7 @@ import { css, cx } from 'emotion';
 import tinycolor from 'tinycolor2';
 
 import { Themeable, withTheme, getLogRowStyles, Icon } from '@grafana/ui';
+import { LogMessageAnsi } from '@grafana/ui/src/components/Logs/LogMessageAnsi';
```

Comment:
Can you please export this component from grafan-ui and add it to the import above?

Reply:
Done!